src/components/Button/__tests__/Button.spec.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import {
+  expect,
+  test,
+} from '@playwright/experimental-ct-react';
+import {
+  mixPropTests,
+  propTests,
+} from '../../../../tests/playwright';
+import {
+  ButtonForTest,
+  ButtonForRefTest,
+} from './Button.story';
+
+test.describe('Button', () => {
```

Comment:
@mbohal @adamkudrna I would suggest starting the review from here. This is complex example representing visual testing, including `mixPropTests` that creates combinations of multiple prop tests.